assets/src/dashboard/utils/usePagePreviewSize.js

```diff
@@ -23,9 +23,22 @@ import { useDebouncedCallback } from 'use-debounce';
  * Internal dependencies
  */
 import theme from '../theme';
-import { DASHBOARD_LEFT_NAV_WIDTH, PAGE_RATIO, WPBODY_ID } from '../constants';
+import {
+  DASHBOARD_LEFT_NAV_WIDTH,
+  PAGE_RATIO,
+  WPBODY_ID,
+  TWO_THIRDS_RATIO,
+} from '../constants';
 import { useResizeEffect } from './';
 
+export const getHeightOptions = (width) => {
+  const fullBleedHeight = width / PAGE_RATIO;
+  const twoThirdsRatio = width / TWO_THIRDS_RATIO;
+
+  const dangerZoneHeight = fullBleedHeight - twoThirdsRatio;
+
+  return { fullBleedHeight, dangerZoneHeight };
```

Comment:
Sticking with the naming from edit-story, this code would be:
```
const fullBleedHeight = width / FULLBLEED_RATIO;
const height = width / PAGE_RATIO;

const dangerZoneHeight = fullBleedHeight - height;

return { height, dangerZoneHeight };
```

Also, notice that I'm returning `height` instead of `fullBleedHeight`, we want the base height used to be the regular height, so the full card height will be `height + dangerZoneHeight`.

Reply:
I feel so bad that I took up your time reviewing this draft! So the update I was working on this AM is so that we are returning ` { fullBleedHeight, storyHeight };` and not doing any subtraction here, just using the two heights per Dima's explanation in slack: 

<img width="315" alt="Screen Shot 2020-06-17 at 12 16 04 PM" src="https://user-images.githubusercontent.com/10720454/84940195-5707d180-b094-11ea-8812-010a4a372be7.png">

Reply:
Ha, no problem.  I know my workflow for PR's is that I normally assign the reviewers and then click "Ready for review", this way github doesn't auto-assign random people to review it.  So I thought maybe you followed that procedure and forgot to hit "Ready for review" (which I've myself have done :D)... Also, I wasn't in the slack thread where you mentioned you were going back to `draft` mode, I only say the lines where "danger zone" was stuck in your head :D 

But it's all good.  I'll definitely respect the draft mode for your PRs next time and not give you feedback you already know 😎

Reply:
nopeee you're right! i had swapped back to draft mode and then forgot to unassign homies. You had helpful feedback regardless, I just know you're busy!